src/components/AuctionModal/AuctionModal.tsx

```diff
@@ -45,38 +47,21 @@ const AuctionModal: React.FC<AuctionModalProps> = ({ open, onClose, photo, onNex
 
 					socket.on(`new_bid`, (bidMessage: BidResponseDto & { room: string }) => {
 						if (bidMessage?.room === roomName) {
-							fetchInitialData();
+							fetchAuctionPhotoData();
 						}
 					});
 
 					socket.on(`finish_auction`, (message: { aucitonPhoto: AuctionPhotoDto; room: string }) => {
 						onClose();
 
 						if (message?.room === roomName) {
-							if (user?.id === message.aucitonPhoto.winner_user_id_1)
-								toastSuccess("Congratulations! You won the auction, make payment to photo as soon as possible to buy it.", {
-									position: "bottom-center",
-									autoClose: false,
-									closeOnClick: false,
-								});
+							if (user?.id === message.aucitonPhoto.winner_user_id_1) toastSuccess("Congratulations! You won the auction, make payment to photo as soon as possible to buy it.");
 							else if (user?.id === message.aucitonPhoto.winner_user_id_2)
-								toast("You are the second winner. If the first winner does not purchase the photo from the auction, you would buy it.", {
-									position: "bottom-center",
-									autoClose: false,
-									closeOnClick: false,
-								});
+								toast("You are the second winner. If the first winner does not purchase the photo from the auction, you would buy it.");
 							else if (user?.id === message.aucitonPhoto.winner_user_id_3)
-								toast("You are the third winner. If the second winner does not purchase the photo from the auction, you would buy it.", {
-									position: "bottom-center",
-									autoClose: false,
-									closeOnClick: false,
-								});
-							else
-								toastWarning("You cannot win the auction.", {
-									position: "bottom-center",
-									autoClose: false,
-									closeOnClick: false,
-								});
+								toast("You are the third winner. If the second winner does not purchase the photo from the auction, you would buy it.");
+							else if (!myPhotos.some((photo) => photo.id === dataAuctionPhoto.photo_id)) toastWarning("You cannot win the auction.");
```

Comment:
```suggestion
							else if (!myPhotos.some((photo) => photo.id === dataAuctionPhoto.photo_id)) toastWarning("You did not win the auction.");
```

bence böyle daha mantıklı, cannot deyince direkt hiç "izin" vermemişiz gibi falan bir anlam çıkıyor bence

Reply:
matıklı